sdk/core/System.BinaryData/src/ReadOnlyMemoryStream.cs

```diff
@@ -126,5 +144,29 @@ private static void ValidateReadArrayArguments(byte[] buffer, int offset, int co
                 throw new ArgumentOutOfRangeException(nameof(count));
             }
         }
+
+        private void ValidateNotClosed()
+        {
+            if (!_isOpen)
+            {
+                throw new ObjectDisposedException(null, "Cannot access a closed Stream");
+            }
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            try
+            {
+                if (disposing)
+                {
+                    _isOpen = false;
+                    _content = null;
```

Comment:
```suggestion
                    _content = default;
```

`_content = null;` is actually `_content = (ReadOnlyMemory<byte>)(byte[])null;`

Assigning it to default (or the more verbose equivalent `ReadOnlyMemory<byte>.Empty`) is a little less work and a bit less confusing (as to how null was assigned to a value type)
`null`